syncer/peers/nofork/manager.go

```diff
@@ -0,0 +1,241 @@
+package nofork
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/0xPolygon/polygon-edge/blockchain"
+	"github.com/0xPolygon/polygon-edge/network/grpc"
+	"github.com/0xPolygon/polygon-edge/syncer/peers"
+	"github.com/0xPolygon/polygon-edge/syncer/peers/nofork/proto"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+const (
+	SyncPeersManagerLoggerName = "sync-peers-manager"
+)
+
+type SyncPeersManager struct {
+	logger     hclog.Logger
+	network    Network
+	blockchain Blockchain
```

Comment:
Please note that by far:
* the outer syncer obj has references to these modules
* the manager obj has references to these modules
* the service server obj has references to these modules

And they all reference each other in some way 🤷‍♂️ 

cc @dbrajovic

Reply:
@zivkovicmilos I don't see what you wanted to say. What is your concern?